secure/key-manager/src/counters.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_secure_push_metrics::{define_counters, Counter};
+use once_cell::sync::Lazy;
+use std::sync::Arc;
+
+// Metrics for the key manager. Uses the "libra_key_manager" prefix for all counters.
+define_counters![
+    "libra_key_manager",
+    (
+        full_consensus_rotations: Counter,
+        "counts the number of full consensus key rotations performed by the key manager"
+    ),
+    (
+        rotation_transaction_resubmissions: Counter,
+        "counts the number of times the key manager had to resubmit a rotation transaction to the blockchain"
+    ),
+    (
+        no_actions_required: Counter,
+        "counts the number of times the key manager determined that no actions were required"
+    ),
+    (
+        reconfiguration_waits: Counter,
```

Comment:
same

waiting_on_reconfiguration

Reply:
Specific to the consensus key (i.e., waiting on a reconfiguration event to sync up the consensus key between the ValidatorConfig and ValidatorInfo). I've updated the name to "waiting_on_consensus_reconfiguration"